packages/backend/src/modules/task/task.service.ts

```diff
@@ -28,6 +28,10 @@ export class TaskService {
   findById(_id: string): Promise<Task> {
     return this._taskRepository.findById(_id)
   }
+
+  findByIdAndRemove(_id: string): Promise<Task> {
+    return this._taskRepository.findByIdAndRemove(_id)
```

Comment:
you should also remove task from DashboardRepository. Check dashboard model. We use code duplications

Reply:
I dunno why we keep the same things in two different places 😕